src/processor/connector/snapshot_arm_template.py

```diff
@@ -35,11 +35,21 @@ def invoke_az_cli(args_str):
     """ 
     Invoke azure cli command
     """
+    login_user = os.environ.get('AD_LOGIN_USER', None)
+    login_password = os.environ.get('AD_LOGIN_PASSWORD', None)
+
+    if not login_user or not login_password:
+        logger.error("`loginUser` or `loginPassword` field is not set in environment")
+        return {"error" : "`loginUser` or `loginPassword` field is not set in environment"}
+        
+    os.system("az login -u " + login_user + " -p " + login_password)
```

Comment:
Add  azexe path also.
azexe = os.environ.get('AZEXE', 'az')
and then add in all places where az command is used.